.gitignore

```diff
@@ -30,3 +30,49 @@ vcpkg
 
 # Visual Studio Code
 .vscode/
+
+# Firestore testapp
+firestore/testapp/**/*.dll
```

Comment:
Drive-by comment: Consider just creating a .gitignore in the `firestore` subdirectory, rather than putting Firestore-specific entries in the top-level .gitignore.